README.rst

```diff
@@ -242,6 +242,30 @@ This script will go through the process of performing the following tasks
    * openstack configuration
    * tempest testing
 
+**Note**: When setting up a multi-node on a cloud which doesn't have access to test pulp (i.e. everywhere except SMS lab) a separate local pulp must be deployed. Before doing so, it is a good idea to make sure your seed VM has sufficient disk space by setting ``seed_disk_size`` in your ``terraform.tfvars`` to an appropriate value (100-200 GB should suffice). In order to set up the local pulp service on the seed, first obtain/generate a set of Ark credentials, then add the following configuration to ``etc/kayobe/environments/ci-multinode/stackhpc-ci.yml``
+
+.. code-block:: console
+
+   stackhpc_release_pulp_username: <ark-credentials-username>
+   stackhpc_release_pulp_password: !vault |
+          <vault-encrypted-ark-password>
```

Comment:
Might be worth a line just explaining how to vault encrypt things. It's the sort of thing that a client might want to do but wouldn't have experience in

Reply:
Client's can't run the workflow to generate credentials so unlikely that they'd be doing this step

Reply:
But they could use their existing client creds for their cloud

Reply:
Actually, release pulp pasword/username should already be defined for clients